Cargo.toml

```diff
@@ -128,3 +128,11 @@ rpath = false
 # it to actually happen when running tests with --release
 lto = true
 opt-level = 3
+
+# The profile is needed for faster linking in case we need to run locally a small amount of tests or just test
+# buisness logic rather than test gas cost. E.g. of using the profile with the cargo:
+# `cargo test --profile fast-link --features mainnet-test name_of_test_we_want_execute`
+[profile.fast-link]
+inherits = "dev"
+opt-level = 0
+lto = false
```

Comment:
About the `rustflags`, I  guess they're redundant because they are more specific settings, which could cause some problems with building in the future. Shoutly, it isn't worth it.

And it looks like the `lto = false` is faster than `lto = "thin"` because `lto = false` with `opt-level = 0` [doesn't use](https://doc.rust-lang.org/cargo/reference/profiles.html#lto) `lto` at all.